colophon.php

```diff
@@ -37,11 +37,11 @@ function team51_credits( $args = array() ) {
 		)
 	);
 
-	$credit_links = array();
+	$credit_links   = array();
+	$partner_domain = wp_parse_url( get_site_url(), PHP_URL_HOST );
```

Comment:
I know it's not very likely, but should we validate the result of the `wp_parse_url` call in case there's an unexpected error and it returns false?

Reply:
I have no objection to this, but ... just change the referrer to https://wpspecialprojects.wordpress.com/ maybe if it doesn't come back as something legit so there's still some attribution?

Reply:
I added a fallback in case the `wp_parse_url` returns `false`